### ISSUE TITLE
Importance of "order" argument

### ISSUE THREAD

**Issue by samlobe** [varun-shanker/structural-evolution#6]
Hi,
This is very nice work! I have one technical question: 
I noticed that ESM's developers have a multichain_util.py that mentions, "For best performance, put the **target chain** first in concatenation", and then proceeds to force the target chain coords to be first (see their multichain_util._concatenate_coords function).
However your multichain_util.py specifically allows either chain to be first with the "order" argument. 
So you allow either the antibody or antigen to be first while ESM forces us to put the antibody first in scoring. 
In other words, ESM-IF does not consider the antigen sequence when scoring log-likelihoods for the antibody, but you do. And this matters in an autoregressive model like ESM-IF.
 
Do you have a reason for this? I assume ESM's developers did some performance testing where they got better results by putting the target chain (e.g. antibody) first, so I'm inclined to stick with their suggestion.
